### PR TITLE
feat(cli): add temporary docs link to platform cli help

### PR DESCRIPTION
**Pull request by sampolahtinen**

### CODE REVIEW

File: packages/cli/src/platform/$.ts [hunk 1]
```diff
@@ -24,7 +25,10 @@ export class $ implements Command {
       ['--token', '', 'Specify a token to use for authentication'],
     ],
     examples: ['prisma platform login', 'prisma platform project create --workspace=<id>'],
-    additionalContent: ['For detailed command descriptions and options, use `prisma platform [command] --help`'],
+    additionalContent: [
+      'For detailed command descriptions and options, use `prisma platform [command] --help`',
+      `For additional help or feedback, visit ${underline('https://pris.ly/platform-cli-docs')}`,
```
[the remark this answers is not in the document]
Comment (Jolg42) [reply packages/cli/src/platform/$.ts:30]: I'm not sure where the "feedback" can be given there, it's only docs to me.

**Comment by Jolg42** [comment]
Note: you'll need to update the platform test snapshot with `-u` before we can merge.

**Comment by tatianainama** [comment]
@sampolahtinen Do you think we could include this link in the `hasHelpFlag` utility? so instead of:
```
✦ prisma (main) ✧ pn --F prisma prisma platform project -h --early-access

Coming soon: help output for this command.
```

we could have something like:
```
✦ prisma (main) ✧ pn --F prisma prisma platform project -h --early-access

Help output for this command will be available soon. In the meantime, visit https://pris.ly/cli/platform-docs for more information.
```

File: packages/cli/src/__tests__/commands/platform/Platform.test.ts [hunk 2]
```diff
@@ -60,6 +60,7 @@ describe('--early-access flag', () => {
         $ prisma platform project create --workspace=<id>
 
       For detailed command descriptions and options, use \`prisma platform [command] --help\`
+      For additional help visit https://pris.ly/cli/platform-docs
```
Comment (Jolg42) [inline packages/cli/src/__tests__/commands/platform/Platform.test.ts:63]: The link is not working for me, it redirects to prisma.io

**Review by Jolg42 — APPROVED** [review]

File: packages/cli/src/platform/$.ts [hunk 3]
```diff
@@ -24,7 +25,10 @@ export class $ implements Command {
       ['--token', '', 'Specify a token to use for authentication'],
     ],
     examples: ['prisma platform auth login', 'prisma platform project create --workspace=<id>'],
-    additionalContent: ['For detailed command descriptions and options, use `prisma platform [command] --help`'],
+    additionalContent: [
+      'For detailed command descriptions and options, use `prisma platform [command] --help`',
+      `For additional help visit ${underline('https://pris.ly/cli/platform-docs')}`,
```
Comment (Jolg42) [inline packages/cli/src/platform/$.ts:30]: I forgot @sampolahtinen but it's better to use `link()` underline is not enough

**Pull request merged**